tests/NHSD.BuyingCatalogue.Identity.Api.UnitTests/Services/ProfileServiceTests.cs

```diff
@@ -0,0 +1,377 @@
+                .ReturnsAsync(expectedApplicationUser);
+
+            var sut = ProfileServiceBuilder
+                .Create()
+                .WithUserRepository(applicationUserRepositoryMock.Object)
+                .Build();
+
+            var profileDataRequestContext = ProfileDataRequestContextBuilder
+                .Create()
+                .WithSubjectId(expectedApplicationUser.Id)
+                .Build();
+
+            await sut.GetProfileDataAsync(profileDataRequestContext);
+
+            var expected = new List<KeyValuePair<string, string>>
+            {
+                new KeyValuePair<string,string>(Subject, expectedApplicationUser.Id),
+                new KeyValuePair<string,string>(PreferredUserName, expectedApplicationUser.UserName),
+                new KeyValuePair<string,string>(JwtRegisteredClaimNames.UniqueName, expectedApplicationUser.UserName),
+                new KeyValuePair<string,string>(GivenName, expectedApplicationUser.FirstName),
+                new KeyValuePair<string,string>(FamilyName, expectedApplicationUser.LastName),
+                new KeyValuePair<string,string>(Name, $"{expectedApplicationUser.FirstName} {expectedApplicationUser.LastName}"),
+                new KeyValuePair<string,string>(Email, expectedApplicationUser.Email),
+                new KeyValuePair<string,string>(EmailVerified, expectedApplicationUser.EmailConfirmed.ToString(CultureInfo.CurrentCulture).ToLowerInvariant()),
+                new KeyValuePair<string,string>(PrimaryOrganisationId, expectedApplicationUser.PrimaryOrganisationId.ToString()),
+                new KeyValuePair<string,string>(OrganisationFunction, expectedApplicationUser.OrganisationFunction),
+                new KeyValuePair<string,string>(ApplicationClaimTypes.Organisation, "Manage")
+            };
+
+            var actual = profileDataRequestContext.IssuedClaims.Select(item => new KeyValuePair<string, string>(item.Type, item.Value));
+            actual.Should().BeEquivalentTo(expected);
+        }
+
+        [TestCase("SomeId", "SomeUserName", Subject, PreferredUserName, JwtRegisteredClaimNames.UniqueName, PrimaryOrganisationId)]
+        [TestCase("SomeId", null, Subject, PrimaryOrganisationId)]
+        public async Task GetProfileDataAsync_GivenApplicationUserWithId_ReturnExpectedClaimList(
+            string expectedUserId,
+            string expectedUserName,
+            params string[] expectedClaimTypes)
+        {
+            var expectedApplicationUser = ApplicationUserBuilder
+                .Create()
+                .WithId(expectedUserId)
+                .WithUserName(expectedUserName)
+                .Build();
+
+            Mock<IUserRepository> applicationUserRepositoryMock = new Mock<IUserRepository>();
+            applicationUserRepositoryMock.Setup(x => x.FindByIdAsync(It.IsAny<string>()))
+                .ReturnsAsync(expectedApplicationUser);
+
+            var sut = ProfileServiceBuilder
```

Comment:
I hate the variable name 'sut', profileService would be better. We know that's the system under test anyway, and it makes it more readable I think

Reply:
I thought this would help with readability for other developers. You know to always look for sut across all your tests & don't have to deal with the different names across multiple test fixtures.

[System under test, actual & expected](https://medium.com/@lorenzen.jacob/unit-test-naming-conventions-875786f53790)